### DISCUSSION TITLE
Daily Standup (Friday, June 3rd)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#87] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finish #86  (still testing) 
- Wrote some issues

**What are your priorities for the next 24 hours?**:
 - 🍻

**Blockers**:
 - None :)

**Shoutouts**:
 - @CarlosCRG19 for the amazing signup ui

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Post Job Offer testing

**What are your priorities for the next 24 hours?**:
 - Same as @antonieto 

**Blockers**:
 - Getting some documents

**Shoutouts**:
 - @antonieto @BrandonMagana for being my guests

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Restructured the search aspirant form
 - Changed a bunch of styling stuff

**What are your priorities for the next 24 hours?**:
 - Finished it

**Blockers**:
 - There is a code that runs during npm install and is causing the server to crashed.

**Shoutouts**:
 - @DavBot02

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Implement code suggestions for @antonieto 's pr 

**What are your priorities for the next 24 hours?**:
 -  Add new properties to backend models
 -  Pistear

**Blockers**:
 - Time 

**Shoutouts**:
 - @Every One

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on signup context and some custom components, introduced theme and tested endpoints for signup

**What are your priorities for the next 24 hours?**:
 - 75% of signup

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana and @antonieto for their hard work with the prs
